backend/fms_core/template_importer/row_handlers/sample_qc/sample.py

```diff
@@ -39,7 +39,7 @@ def process_row_inner(self, sample, sample_information, process_measurement, pro
                               volume=new_volume,
                               concentration=sample_information['concentration'])
 
-            # Update derived_sample's flags with sample information
+            # Update the sample's flags with sample information
             _, self.errors['flags'], self.warnings['flags'] = update_qc_flags(sample_obj,
                                                                               sample_information['quantity_flag'],
```

Comment:
i would use named parameter here because it's very easy to confuse or intervert the order they are passed to the function.